### PR TITLE
[IMP] base,**: rewrite some error messages

### PR DESCRIPTION
**Pull request by bouvyd**
Maintain the playful tone whilst avoiding nonsensical phrasing.

I will not take comment at this time, thank you.

Task-🤔🙃😬💅😐🤓

### CODE REVIEW

**Comment by bouvyd** [comment]
@robodoo r+ when green darling

Thanks to @ticodoo for the feedback in the PR-that-shall-not-be-linked-to, I did end up using some of it. I did not implement the feedback regarding 'poof' for the following reasons:
- if considered too offensive for UK, can be handled via i18n by translators 🇬🇧 
- i believe context is key, and here there can be little confusion regarding what we mean . As a comparison, we have in french the word 'pouf' which has the same issue; it means 'poof' (as-in the disappearing act) and also 'bitch' (or another word that rhymes with _blunt_), but frankly you never confuse one for the other and seeing it used in a professional setting is not jarring at all (though I know this is a comparison and not an exactly similar situation) 🪄
- if the Fairy Oddparents feel comfortable naming the baby fairy character Poof, how bad can it be? 🧚
Poof, the fairy
- I'm going to assume that UK Gays are funny guys and that it'll make them giggle the same way I giggle whenever I hear a british character asking for a _fag_ 💅
- if someone files an official protest stating how homophobic we are, i'll use my influence to make sure they don't get invited to the Gay Agenda meetings we hold semi-annually to plan for world domination - should calm them down 🌏

Thanks also to ChatGPT who did contribute indirectly and who provided this insightful haiku regarding this matter:
> In chat's subtle flow,
> Fabien's quest for words refined,
> Humor in each line.

**Comment by bouvyd** [comment]
@robodoo retry

**Comment by tde-banana-odoo** [comment]
Even robodoo would like to comment.

**Comment by william-andre** [comment]
Thank you but...

Why are you doing something in the dark at 11pm instead of doing it where everyone asked for it?
Why give us all a middle finger, just to agree with us behind our back?

I'm glad that it is done, but for a PR about communication, I think the (lack of) communication is terrible.

**Comment by dmo-odoo** [comment]
Since the intent of the original PR was controversial as a whole, regardless of the specific content of the messages, reopening the discussion on it would have been an open door for people to throw a spanner at the works using endless debates in an attempt to prevent the PR to be merged altogether.

Since Fabien really believes in this new light-hearted tone despite all the negative feedback received so far, it made sense to merge the original PR first to ensure that the change of tone was definitively applied and, in a second pass, address the specific content of the messages themselves.

Not saying that I agree with the change of tone or with the decision to do it in two separate PR, but you asked "why" and I knew the answer so I thought I would give you the context that you were missing.

**Comment by william-andre** [comment]
Thanks for the reply, but you missed my point.

They acted like they don't care on the original PR. This is what caused all the discussion to span in the first place. So what they were doing was actively keeping the debate open and asking us to ask them to justify it.
**The way they acted is actually the cause of what they say they wanted to avoid.**[^1]

If this PR had been implemented on the original PR when it was asked for, there would never have been such a hard push-back.

Exactly why I said: "Why give us all a middle finger, just to agree with us behind our back?"

[^1]: insert time travel reference here

File: addons/web_editor/static/src/components/media_dialog/image_selector.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
         <t t-if="!hasContent and !isFetching">
             <div t-if="state.needle" class="o_nocontent_help">
                 <p class="o_empty_folder_image">No images found.</p>
-                <p class="o_empty_folder_subtitle">Psst! Want to impress your screen? Upload from the button in the top right corner!</p>
+                <p class="o_empty_folder_subtitle">Wow, it feels a bit empty in here. Upload from the button in the top right corner!</p>
```
Comment (Zinston) [inline addons/web_editor/static/src/components/media_dialog/image_selector.xml:33]: <3

**Comment by oomsveta** [comment]
Hi, Louis (wil) from R&D i18n here. For the record, we have received a ticket for an unfortunate incident involving the "Poof!" wording. I agree with Damien: no human would ever read that as the British slur, right? Well, no human indeed, but not all our readers are human. We overlooked the sensible use case of admins machine-translating Odoo into their own language. Yesterday, someone was really disappointed to be greeted in Odoo 17 with a very bad homophobic slur

**Comment by william-andre** [comment]
And Poof! Reviewers were right! :exploding_head:

**Pull request closed without merging**